internal/build/docker.go

```diff
@@ -370,17 +376,35 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 	return result, nil
 }
 
-func getDigestFromOutput(reader io.Reader) (digest.Digest, error) {
+func getDigestFromBuildOutput(reader io.Reader) (digest.Digest, error) {
 	aux, err := readDockerOutput(reader)
 	if err != nil {
 		return "", err
 	}
 	if aux == nil {
-		return "", fmt.Errorf("getDigestFromOutput: No results found in docker output")
+		return "", fmt.Errorf("getDigestFromBuildOutput: No results found in docker output")
 	}
 	return getDigestFromAux(*aux)
 }
 
+func getDigestFromPushOutput(reader io.Reader) (digest.Digest, error) {
+	aux, err := readDockerOutput(reader)
```

Comment:
fwiw i find it confusing to call this variable "aux" -- it only makes sense if you know the format of the docker output